### PR TITLE
Use a single font family.

### PR DESCRIPTION
**Pull request by ADmad**
This keeps things simple and avoids pulling in additional external resource.
Refs #771.

### CODE REVIEW

**Comment by markstory** [comment]
Using raleway was nice as it pulls part of the CakePHP branding elements from the website/docs into the application.

File: templates/layout/error.php [hunk 1]
```diff
@@ -23,7 +23,6 @@
     </title>
     <?= $this->Html->meta('icon') ?>
 
-    <link href="https://fonts.googleapis.com/css?family=Raleway:400,700" rel="stylesheet">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.css">
```
Comment (dereuromark) [inline templates/layout/error.php:26]: This one is still external right?
So maybe we don't have to remove the railway - if we cant get external resources away anyway.

Comment (ADmad) [reply templates/layout/error.php:26]: I am not sure why `normalize.css` is used through CDN. It too should be added to webroot/css itself.

**Comment by ADmad** [comment]
I don't have a strong opinion on this either way, hence closing.

**Pull request closed without merging**